site/en/docs/extensions/whatsnew/index.md

```diff
@@ -16,57 +16,157 @@ updated: 2021-08-04
+
+Also, APIs that use the [`ChromeSetting`](/docs/extensions/reference/types/#type-ChromeSetting)
+prototype now also support promises. The following APIs are affected by this change.
+
+- [`chrome.privacy`](/docs/extensions/reference/privacy/)
+- [`chrome.accessibilityFeatures`](/docs/extensions/reference/accessibilityFeatures/)
+- [`chrome.proxy`](/docs/extensions/reference/proxy/)
+
+{% endDetails %}
+
+{% Details %}
+{% DetailsSummary %}
+Chrome OS APIs
+{% endDetailsSummary %}
+
+- [`chrome.certificateProvider`](/docs/extensions/reference/certificateProvider)
+- [`chrome.documentScan`](/docs/extensions/reference/documentScan)
+- [`chrome.enterprise.deviceAttributes`](/docs/extensions/reference/enterprise_deviceAttributes)
+- [`chrome.enterprise.networkingAttributes`](/docs/extensions/reference/enterprise_networkingAttributes)
+- [`chrome.fileBrowserHandler`](/docs/extensions/reference/fileBrowserHandler)
+- [`chrome.fileSystemProvider`](/docs/extensions/reference/fileSystemProvider)
+- [`chrome.loginState`](/docs/extensions/reference/loginState)
+- [`chrome.printingMetrics`](/docs/extensions/reference/printingMetrics)
+- [`chrome.wallpaper`](/docs/extensions/reference/wallpaper)
+
+{% endDetails %}
+
+### Chrome 96: `declarativeNetRequestWithHostAccess` permission
+
+The new `declarativeNetRequestWithHostAccess` permission allows existing Manifest V2 extensions that
+leverage `webRequest`, `webRequestBlocking`, and site-specific host permission to migrate to the
+[`chrome.declarativeNetRequest`](/docs/extensions/reference/declarativeNetRequest/) API without
+requiring the end user to approve a new permissions grant.
+
+### 2021-10-27: Review violation troubleshooting updates {: #2021-10-27-reivew-troubleshooting }
 
 The [Troubleshooting Chrome Web Store violations](/docs/webstore/troubleshooting/) page has been
 updated to provide developers with more detailed guidance for common reasons for rejection.
 
-### Manifest V2 support timeline
+### Chrome 95: Inject scripts directly into pages {: #m95-page-script-injection }
+
+The [`chrome.scripting`](/docs/extensions/reference/scripting) API's
+[`executeScript()`](/docs/extensions/reference/scripting/#method-executeScript) method can now
+inject scripts directly into a page's main world. Previously, extensions could only inject directly
+into the extension's isolated world. For more information on isolated worlds, see the documentation
+on [content scripts](/docs/extensions/mv3/content_scripts/#isolated_world).
+
+### Chrome 95: Promise support for Storage API {: #m95-storage-promise-support }
+
+Methods on the Manifest V3 version of the [`chrome.storage`](/docs/extensions/reference/storage/)
+API now return promises.
+
+### Chrome 95: Cross origin isolation opt-in now required {: #m95-coi-required }
+
+Extensions and apps must now opt into [cross origin
+isolation](https://web.dev/cross-origin-isolation-guide/) in order to use features like
```

Comment:
```suggestion
isolation](https://web.dev/cross-origin-isolation-guide/) to use features like
```

Reply:
Dropping this section per [@rdcronin's comment](https://github.com/GoogleChrome/developer.chrome.com/pull/1714/files#r746789544).